sentry-kotlin-multiplatform/src/androidTest/kotlin/io/sentry/kotlin/multiplatform/SentryLevelTest.kt

```diff
@@ -0,0 +1,32 @@
+package io.sentry.kotlin.multiplatform
+
+import org.junit.Test
+import kotlin.test.assertEquals
+import io.sentry.kotlin.multiplatform.extensions.toAndroidSentryLevel
+
+class AndroidSentryLevelTest {
+
+    @Test
+    fun `convert SentryLevel to AndroidSentryLevel has proper value`() {
```

Comment:
It's a good practice to have a test for each case instead all of them.
In case one of them fails, you won't know which one failed unless you debug it.